.github/workflows/windows.yml

```diff
@@ -243,6 +244,16 @@ jobs:
       # Build
       #
 
+      - name: Install build dependencies
+        run: |
+          Invoke-WebRequest https://github.com/ninja-build/ninja/releases/download/v1.11.1/ninja-win.zip -OutFile ninja-win.zip -MaximumRetryCount 10
+          Expand-Archive -Force ninja-win.zip
+          # Add it to the GitHub Path so it would be available in the subsequent steps
+          Add-Content -Path $env:GITHUB_PATH -Value "${{ github.workspace }}/ninja-win"
+
+      - name: Configure Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
what toolchain is used by default? maybe we need to hardcode it explicitly?

Reply:
the latest available 14.41